kuzminskiy_stafeev/graph_logger.hpp

```diff
@@ -0,0 +1,42 @@
+#include <chrono>
+#include <ctime>
+#include <fstream>
+#include <iomanip>
+#include <iostream>
+#include <sstream>
+#include "config.hpp"
+#include "graph.hpp"
+#pragma once
+
+class Logger {
+ public:
+  static Logger* get_logger() {
+    if (NULL == _instance) {
+      _instance = new Logger();
+      _instance->ofs.open(config::kLogFilePath, std::ios::app);
+    }
+
+    return _instance;
+  }
```

Comment:
1) Зачем поинтер?
2) В `hpp` только однострочные методы
2) Мjжно проще
```suggestion
  static Logger& get_logger() {
    static Logger instance;
    return instance;
  }
```